CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0
 
 All notable changes to this project will be documented in this file.
 
+## [2.3.0.pre.1] - UNRELEASED
+### Added
+- Added support for `saved_changes?` in `Aggregate::AggregateStore` for rails 5.
+- Added `saved_changes?` method which returns boolean of whether or not changes were made on the most recent save.
```

Comment:
can we also document `aggregate_attribute_saved_changes` in here? Thats a change to the public API as well it looks (which may be nice to have)